3d_segmentation/swin_unetr_brats21_segmentation_3d.ipynb

```diff
@@ -0,0 +1,1021 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    class AverageMeter(object):
</pre><p> </p><p>We have enhanced tool for similar use case:</p><p>&nbsp;https://github.com/Project-MONAI/MONAI/blob/dev/monai/metrics/cumulative_average.py#L20</p><p> </p><p>Could you please take a look? Maybe you can use it instead of this <code>AverageMeter</code>.</p><p> </p><p>Thanks.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/750/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
I see. Unfortunately, I don't have the resources to re-train the tutorial for only changing the AverageMeter. Since this is not a major component of this tutorial, shall be fine to keep as is ?